.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Question
+labels: 'issue: question'
+about: Ask a question about OpenSSL
+---
+
+<!--
+Thank you for your interest in OpenSSL. If this is your first question,
+please take the time to read the following lines before posting it.
+
+For general questions about *using* OpenSSL:
+
+    If you have questions about how to use OpenSSL for specific tasks
+    or how to solve certain problems you have when using it, you might
+    want to ask them on the openssl-users@openssl.org mailing list.
+    There you can get help from a great community of OpenSSL users,
+    not only (but including) the OpenSSL developers. For more information
+    about our mailing lists, see
+    https://www.openssl.org/community/mailinglists.html.
+
+For questions related to build issues:
+
+    Please use the 'Bug report' template.
+
+For other questions:
+
+    Please describe your problem as concise as possible while giving
```

Comment:
Typo: concisely

Reply:
Fixed in 6ac8a120d7e3651a1b7e14cb458280582189d555.